src/Commands/Databases/Make.php

```diff
@@ -27,6 +28,7 @@ public function configure()
             ->addArgument('server', InputArgument::REQUIRED, 'The id of the server to create the database on.')
             ->addOption('user', null, InputOption::VALUE_REQUIRED, 'The username of an (optional) database user to create.', null)
             ->addOption('password', null, InputOption::VALUE_REQUIRED, 'The password of the database user. Required with "user".', null)
+            ->addOption('name', null, InputOption::VALUE_REQUIRED, 'The name of the database user.', null)
```

Comment:
This is actually the name of the database to create, not the name of the database user 🙂